src/riak_kv_stat.erl

```diff
@@ -287,6 +287,27 @@ do_update(ngrrepl_object) ->
     ok = exometer:update([?PFX, ?APP, node, puts, ngrrepl_object], 1);
 do_update(ngrrepl_error) ->
     ok = exometer:update([?PFX, ?APP, node, puts, ngrrepl_error], 1);
+do_update({ttaaefs, all_check}) ->
```

Comment:
Not looking deep, do we have tests that cover functionality or should I script to look for spelling mistakes or something?

Reply:
The riak_test test checks the stats .  If you look at the get_stats function and its use in https://github.com/basho/riak_test/blob/mas-i1804-peerdiscovery/tests/nextgenrepl_rtq_peerdiscovery.erl.

There is also a verify_riak_stats test which gets updated, which just checks that all things are named correctly, and there are no unexpected entries:  https://github.com/basho/riak_test/blob/mas-i1804-peerdiscovery/tests/verify_riak_stats.erl

If this looks like it is covering everything OK, then we're fine.  However, if anything jumps out as looking weird, flag it, and I will add some coverage to the test.